recipe-client-addon/test/browser/browser_NormandyDriver.js

```diff
@@ -1,5 +1,38 @@
 "use strict";
 
+Cu.import("resource://shield-recipe-client/lib/NormandyDriver.jsm", this);
+Cu.import("resource://shield-recipe-client/lib/Storage.jsm", this);
+
+
+// Storage durability checks occur when drivers create new Storage instances.
+// This test determines if `skipDurabilityCheck` correctly ignores the
+// store durability.
+add_task(async function(){
+  const fakeSandbox = { Promise };
+  const driver = new NormandyDriver({ sandbox: fakeSandbox });
+
+  const store = Storage.makeStorage(Storage.DURABILITY_NAMESPACE, fakeSandbox);
+  await store.setItem(Storage.DURABILITY_KEY, -1);
+  let hadError = false;
+
+  try {
+    // Create storage with the 'skipDurabilityCheck' flag set to true
+    await driver.createStorage('test', true);
```

Comment:
I don't see where `Driver::createStorage` takes a second parameter for skipDurabilityCheck. Is this test stale?